### PR TITLE
README: Adding ExternalProject_Add example

### PR DESCRIPTION
**Pull request by thopiekar**
Added a short description how I added AppImageKit in one of my projects.

### CODE REVIEW

**Comment by probonopd** [comment]
@TheAssassin as our CMake artist in residence, does this "speak" to you?
(For me, the many variables that CMake seems to use everyone makes it hard to understand what it is doing.)

**Comment by TheAssassin** [comment]
Looks quite a bit cryptic to me, too. I'll have a detailed look next week.

**Comment by thopiekar** [comment]
Oops, I thought you merged the PR already. I forgot you have permissions to push into my branches if they are used for PRs 😅

**Comment by TheAssassin** [comment]
So, I suppose this is about building appimagetool and using it in your project, rather than using our (relatively new) libappimage? I'll clarify that a bit then in the README.

**Comment by thopiekar** [comment]
Ok, feel free to do that. No, I'm not using the libappimage, but I guess it should be easy to prevent CMake to generate Makefiles, which build only libappimage 😉

File: README.md [hunk 1]
```diff
@@ -233,3 +233,21 @@ cd AppImageKit/
...
+if(BUILD_OS_LINUX)
+    ExternalProject_Add(AppImageKit
+        GIT_REPOSITORY https://github.com/AppImage/AppImageKit.git
+        GIT_TAG appimagetool/master
+        GIT_SUBMODULES ""
+        CMAKE_COMMAND ${CMAKE_COMMAND} -E env "PATH=${CMAKE_INSTALL_PREFIX}/bin/:$ENV{PATH}" "LD_LIBRARY_PATH=${CMAKE_INSTALL_PREFIX}/lib/" "LDFLAGS=-L${CMAKE_INSTALL_PREFIX}/lib" "CPPFLAGS=-I${CMAKE_INSTALL_PREFIX}/include"  ${CMAKE_COMMAND}
+        CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX} -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_PREFIX_PATH=${CMAKE_INSTALL_PREFIX} -DBUILD_SHARED_LIBS=ON -DBUILD_TESTING=ON
+    )
+
+    SetProjectDependencies(TARGET AppImageKit DEPENDS glib libfuse cairo OpenSSL patch sed wget)
```
Comment (azubieta) [inline README.md]: This doesn't seems to be a native CMake function. I suppose it's a custom macro.
In order to allow other user to reuse you code snippet you must only use native functions.

**Comment by thopiekar** [comment]
Yes, SetProjectDependencies is custom and the BUILD_OS_LINUX variable as
well.

**Comment by TheAssassin** [comment]
This PR has been closed by the original author. No need for reviews or anything.

**Pull request closed without merging**